src/transformers/activations_tf.py

```diff
@@ -63,6 +63,12 @@ def gelu_fast(x):
     return 0.5 * x * (1.0 + tf.tanh(x * coeff2 * (1.0 + coeff1 * x * x)))
 
 
+def quick_gelu(x):
```

Comment:
I really want to delete all of these approximate functions and just not support TF <= 2.3! (But let's leave that for another PR)